src/Core/Util/ColorBrightnessCalculator.php

```diff
@@ -60,7 +60,7 @@ private function calculate($hexColor)
         $hexColor = str_replace('#', '', $hexColor);
 
         if (strlen($hexColor) === 3) {
-            $hexColor .= $hexColor;
+            $hexColor = $hexColor[0] . $hexColor[0] . $hexColor[1] . $hexColor[1] . $hexColor[2] . $hexColor[3];
```

Comment:
Shouldn't it be like
```suggestion
            $hexColor = $hexColor[0] . $hexColor[0] . $hexColor[1] . $hexColor[1] . $hexColor[2] . $hexColor[2];
```

?

Reply:
Sorry, I missed that.

Reply:
No worries it's a small typo it can happen, thank you for your contribution !